src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -1340,6 +1340,14 @@
 			}
 			return rootElement === element || rootElement.contains(element);
 		}
+
+		public setCursor( cssCursor: string) {
+			const unoBody = document.getElementById("uno-body");
+
+			if (unoBody) {
+				unoBody.style.cursor = cssCursor;
```

Comment:
The biggest problem I see with this approach is it won't work for controls deriving from `ButtonBase` because of this css: https://github.com/nventive/Uno/blob/master/src/Uno.UI.Wasm/WasmCSS/Uno.UI.css#L46

The goal was to set the pointer as "pointer" for buttons to improve usuability of such controls in wasm. So changing it globally on the _window_ won't affect buttons and other controls deriving from `ButtonBase`.

Maybe we should also add a property to the root element to temporary deactivate the custom css on buttons while the pointer is _customized_.

Reply:
Using PointerExited / PointerEntered in button code would work ( but then you have to handle it cleverly.. ) [UWP toolkit](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI/Extensions/Mouse/Mouse.cs) provide a way to solve it. 

For my use case, having the property to work everywhere except on button / hyperlink would be ok 

else, spawning a style node and removing it when done would work i guess: 
https://codepen.io/anon/pen/gEbKPB